src/platform_impl/macos/window.rs

```diff
@@ -1253,6 +1253,16 @@ impl UnownedWindow {
     let state = self.shared_state.lock().unwrap();
     state.current_theme
   }
+
+  pub fn set_content_protection(&self, enabled: bool) {
+    unsafe {
+      if enabled {
+        let _: () = msg_send![*self.ns_window, setSharingType: 0];
+      } else {
+        let _: () = msg_send![*self.ns_window, setSharingType: 1];
+      }
```

Comment:
I don't know if this will work with the macro but here goes the suggestion anyways:
```suggestion
	let _: () = msg_send![*self.ns_window, setSharingType: !enabled as i32];
```
or 
```suggestion
	let enabled = !enabled as i32;
	let _: () = msg_send![*self.ns_window, setSharingType: enabled ];
```
or 
```suggestion
	let enabled = if enabled { 0 } else { 1 };
	let _: () = msg_send![*self.ns_window, setSharingType: enabled ];
```